foundationdb/src/tuple/element.rs

```diff
@@ -30,6 +32,8 @@ impl<'a> Element<'a> {
             Element::Tuple(v) => Element::Tuple(v.into_iter().map(|e| e.into_owned()).collect()),
             #[cfg(feature = "uuid")]
             Element::Uuid(v) => Element::Uuid(v),
+            #[cfg(feature = "num-bigint")]
+            Element::BigInt(v) => Element::BigInt(v),
```

Comment:
Could you add a `as_versionstamp` conversion below. That would be really useful.

Reply:
ok will do.